docs/index.md

```diff
@@ -1487,20 +1488,22 @@ A typical setup might look something like the following, where we call `mocha.se
 <head>
   <meta charset="utf-8">
   <title>Mocha Tests</title>
-  <link href="https://unpkg.com/mocha@5.2.0/mocha.css" rel="stylesheet" />
+  <link href="https://unpkg.com/mocha@6.0.0/mocha.css" rel="stylesheet" />
 </head>
 <body>
   <div id="mocha"></div>
 
   <script src="https://unpkg.com/chai/chai.js"></script>
-  <script src="https://unpkg.com/mocha@5.2.0/mocha.js"></script>
+  <script src="https://unpkg.com/mocha@6.0.0/mocha.js"></script>
```

Comment:
if I were you I'd actually just eliminate the version altogether.  it should pull the latest one.  since we are doing a prerelease, this number will be `v6.0.0-0` anyway, and I'm pretty sure we don't want to keep changing it

Reply:
(https://unpkg.com/mocha/mocha.js is just a redirect to the latest version)

Reply:
done. would have removed the version earlier, but thought it might be intentional.